students/hyska_monika/lesson_01_basics/PreviousAndNext.py

```diff
@@ -0,0 +1,9 @@
+# This program returns previous and next numbers for integer number
+
+a = int(input("Put integer number: "))
+
+prev = a - 1
+next = a + 1
```

Comment:
`next` is a built-in, try to avoid those (by qualifying for example `next_integer` `prev_integer`).